Directory.Build.targets

```diff
@@ -1,7 +1,15 @@
 <Project>
 
+  <PropertyGroup>
+    <IsSourceProject Condition="'$(IsSourceProject)' == '' and
+                                '$(IsTestProject)' != 'true' and
+                                '$(IsTestUtilityProject)' != 'true'">true</IsSourceProject>
+  </PropertyGroup>
+
+  <Import Project="$(RepositoryEngineeringDir)unsetispackable.targets" Condition="'$(UnsetIsPackableForNonSourceProjects)' == 'true' and '$(IsSourceProject)' != 'true'" />
   <Import Project="$(RepositoryEngineeringDir)resources.targets" />
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
   <Import Project="$(RepositoryEngineeringDir)testing\runsettings.targets" Condition="'$(IsTestProject)' == 'true'" />
   <Import Project="$(RepositoryEngineeringDir)placeholderpackaging.targets" Condition="'$(IsPlaceholderTargetFramework)' == 'true'" />
   <Import Project="$(RepositoryEngineeringDir)packaging.targets" Condition="'$(IsPackable)' == 'true'" />
```

Comment:
Lines 30-33 are still defining SkipTests which neither the yaml or the traversal build project are using anymore.

Reply:
The `SkipTests` property is still used in other places: https://github.com/search?q=repo%3Adotnet%2Fmaintenance-packages%20SkipTests%20&type=code

Reply:
Right. It's used in the lines I mentioned. What I am asking is that if we're going to be using Test from now on for the build command, why don't we update the rest of the places where SkipTests was used and change them to Test?

Reply:
The `Test` property is an Arcade setting that doesn't flow down into this repository:
- https://github.com/dotnet/arcade/blob/39290b60a9015b444a1762d67e26f0f187eb5cc2/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L122C8-L122C30
- https://github.com/dotnet/arcade/blob/39290b60a9015b444a1762d67e26f0f187eb5cc2/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L130

`/p:Test=false` tells Arcade to not invoke the `Test` target **on the solution file** in Arcade's Build.proj orchestration project.
`SkipTests` is a repo local property that I wouldn't expect devs to pass in as a global property (CLI argument). That is reponsible for disabling the VSTest target per project.

Reply:
Can you check if we're still defining a value for SkipTests anywhere in the repo after the changes in this PR?

The only place I know was setting a value for it was the yaml files. Now it's being removed from there.

If we shouldn't even expect devs to pass it as a global property, then why are we keeping SkipTests for? In what cases will it be useful?

Reply:
You're right. Nevermind.